NOTICE

```diff
@@ -1,10 +1,11 @@
-Copyright 2018-2019 IOV SAS
+Copyright 2018-2020 IOV SAS
+Copyright 2020 Simn Warta
```

Comment:
I though your name was "Simon". ;)

I'm happy to merge this after you've made a contribution since your departure.

Reply:
Ah, thanks. Typing without reading is not a good idea.